unity/Assets/Scripts/CollisionListener.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using UnityStandardAssets.Characters.FirstPerson;
+
+public class CollisionListener : MonoBehaviour
+{
+    //references to the joints of the mid level arm
+    [SerializeField]
+    private bool CascadeCollisionEventsToParent = true;
+    //track what was hit while arm was moving
+    public class StaticCollision
+    {
+        public GameObject gameObject;
+
+        public SimObjPhysics simObjPhysics;
+
+         //indicates if gameObject a simObject
+        public bool isSimObj {
+            get { return simObjPhysics != null; }
+        }
+    }
+    private StaticCollision staticCollided;
```

Comment:
I think we can get rid of this.  It doesn't appear to be used.

Reply:
agreed, this was from the previous version of the logic so should no longer be needed

Reply:
Done, sneaked past me